CSSampleApp/src/main/res/layout/content_main.xml

```diff
@@ -102,14 +102,23 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/label_masking_scenarios" />
 
+        <TextView
+            android:id="@+id/label_custom_variables"
+            style="@style/ListText"
+            android:layout_marginStart="32dp"
+            android:onClick="openCustomVariables"
+            android:text="@string/label_custom_variables"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/title_dynamic_variables" />
+
         <TextView
             android:id="@+id/label_dynamic_variables"
             style="@style/ListText"
             android:layout_marginStart="32dp"
             android:onClick="openDynamicVariables"
             android:text="@string/label_dynamic_variables"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/title_dynamic_variables" />
+            app:layout_constraintTop_toBottomOf="@+id/label_custom_variables" />
```

Comment:
I am not sure that this needs to be changed.

Reply:
Yes, because I inserted the CVARs between the title and the DVARs. I pushed another commit where I changed the ID of the title.